Bottom: 48d970328442a027b014905c2db5b3d56fd676f4
Top:    e4fdf3df863d148c8e74af8a6f7ea495adc42a0c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-02-25 19:43:52 -0500

Win32: Disable the SpaceNavigator code


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 38f8a88..d40dd86 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -552,8 +552,8 @@ ghid_port_window_enter_cb (GtkWidget * widget,
       /* Make sure drawing area has keyboard focus when we are in it.
        */
       gtk_widget_grab_focus (out->drawing_area);
-      if (ghidgui->snavi != NULL)
-        snavi_set_led (ghidgui->snavi, TRUE);
+      //if (ghidgui->snavi != NULL)
+      //  snavi_set_led (ghidgui->snavi, TRUE);
     }
   ghidgui->in_popup = FALSE;
 
@@ -586,8 +586,8 @@ ghid_port_window_leave_cb (GtkWidget * widget,
       return FALSE;
     }
 
-  if (out->has_entered && !ghidgui->in_popup && ghidgui->snavi != NULL)
-    snavi_set_led (ghidgui->snavi, FALSE);
+  //if (out->has_entered && !ghidgui->in_popup && ghidgui->snavi != NULL)
+  //  snavi_set_led (ghidgui->snavi, FALSE);
 
   out->has_entered = FALSE;
 
diff --git a/src/hid/gtk/snavi.c b/src/hid/gtk/snavi.c
index 2f58f02..d5c5850 100644
--- a/src/hid/gtk/snavi.c
+++ b/src/hid/gtk/snavi.c
@@ -12,6 +12,7 @@
  *
  */
 
+#if 0
 #include <stdio.h>
 #include <errno.h>
 #include <stdlib.h>
@@ -21,9 +22,11 @@
 
 #include <linux/types.h>
 #include <linux/input.h>
+#endif
 
 #include <glib.h>
 
+#if 0
 #define test_bit(bit, array)  (array [bit / 8] & (1 << (bit % 8)))
 
 static void (*update_pan_cb)(int, int, int, gpointer);
@@ -150,6 +153,7 @@ gboolean snavi_event (GIOChannel   *source,
 
   return TRUE;
 }
+#endif
 
 
 GIOChannel *
@@ -159,6 +163,9 @@ setup_snavi (void (*update_pan)(int, int, int, gpointer),
              void (*button)(int, int, gpointer),
              gpointer data)
 {
+  return NULL;
+
+#if 0
   GIOChannel *snavi;
 
   update_pan_cb = update_pan;
@@ -186,4 +193,5 @@ setup_snavi (void (*update_pan)(int, int, int, gpointer),
     }
 
   return snavi;
+#endif
 }
